index.html

```diff
@@ -180,6 +185,22 @@
       </p>
 
       </section>
+
+      <section>
+      <h3>Processing incoming payment method data</h3>
+
+      <p>When the PaymentRequest API is to pass data to a payment app supporting basic-card as part
+      of the <a>paymentRequest.show()</a> algorithm, the payment app must process any non-null
```

Comment:
Chatting with @marcoscaceres on the editor call, I think there are two issues here:

* We should remove any language around payment apps from this section, as payment apps aren't part of the .show algorithm

* We should throw an exception if we see basic-card and things that aren't supported as a part of that (e.g. 'supportedTypes: ["banana"]')